react-common/components/share/Kiosk.ts

```diff
@@ -1,20 +1,17 @@
 export const addGameToKioskAsync = async (kioskId: string, gameId: string) => {
     const updateKioskUrl = "https://makecode.com/api/kiosk/updatecode";
-    try {
-        const response: Response = await fetch(updateKioskUrl, {
-            method: 'POST',
-            headers: {
-                'Accept': 'application/json',
-                'Content-Type': 'application/json'
-            },
-            body: JSON.stringify({
-                "kioskCode": kioskId,
-                "shareId": gameId
-            }),
-        });
-        await response.json();
-    }
-    catch (error) {
-        throw new Error(error.message);
+    const response: Response = await fetch(updateKioskUrl, {
+        method: 'POST',
+        headers: {
+            'Accept': 'application/json',
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+            "kioskCode": kioskId,
+            "shareId": gameId
+        }),
+    });
+    if (!response.ok) {
+        throw new Error(lf(response.statusText));
```

Comment:
Do we actually want lf here? It looks like we're comparing it to a hard-coded "Not Found" string later on, so I'd guess we don't want to localize it (in which case that check would break).

Reply:
(Assuming this is what becomes error.message in the catch section in ShareInfo)

Reply:
whoops good catch here, shouldn't be `lf` here anyways as it's not static text (could use `rlf`, but it's unlikely the error message would actually be in our set of translations so might as well just return the text directly)